ui/apps/platform/src/Containers/Network/Header/NetworkSearch.js

```diff
@@ -27,12 +28,19 @@ function NetworkSearch({
         }
     }
 
-    let prependAutocompleteQuery;
+    let prependAutocompleteQuery = [];
```

Comment:
At the risk of expanding the scope of the change, this improvement brings to mind another possible step:

```js
    const orchestratorComponentShowState = localStorage.getItem(ORCHESTRATOR_COMPONENT_KEY);
    const prependAutocompleteQuery =
        orchestratorComponentShowState === 'true' ? [] : [...orchestratorComponentOption];
```

Despite what you might expect from the name `orchestratorComponentOption` corresponds to default false.

Reply:
Outside the scope of this pull request, the option seems relevant for **Namespaces** filter:

Hide does not render **kube-system** namespace
![hide](https://user-images.githubusercontent.com/11862657/161788054-91baac61-f31e-4afe-a443-0395add193f7.png)

Show does render **kube-system** namespace
![show](https://user-images.githubusercontent.com/11862657/161788082-7ea16a29-26b8-4574-8974-d30495cfe8ba.png)